cfme/tests/test_login.py

```diff
@@ -39,17 +41,20 @@ def test_login(context, method, appliance):
 @pytest.mark.tier(2)
 @pytest.mark.sauce
 @pytest.mark.parametrize('context', [ViaUI])
+# BZ 1632718 is only relevant for Chrome browser
+@pytest.mark.meta(blockers=[BZ(1632718, forced_streams=['5.10'])])
 def test_bad_password(context, request, appliance):
     """ Tests logging in with a bad password. """
 
     username = conf.credentials['default']['username']
     password = "badpassword@#$"
     cred = Credential(principal=username, secret=password)
     user = appliance.collections.users.instantiate(credential=cred, name='Administrator')
-    if appliance.version.is_in_series('5.7'):
-        error_message = "Sorry, the username or password you entered is incorrect."
-    else:
-        error_message = "Incorrect username or password"
+
+    error_message = VersionPicker({
+        version.LOWEST: "Incorrect username or password",
+        '5.10': "Login failed: Unauthorized"
+    }).pick(appliance.version)
```

Comment:
error_message = VersionPicker({
        LOWEST: "Incorrect username or password",
        '5.10': "Login failed: Unauthorized"
    }).pick(appliance.version)

Reply:
OPTIONAL: I'd just use if appliance.version > 5.10. This construction is too sophisticated.